frontend/src/app/app.test.tsx

```diff
@@ -1,17 +1,18 @@
 import React from 'react';
-import Index from '@app/index';
+import Search from '@app/search';
+import "isomorphic-fetch"
 
 import { mount, shallow } from 'enzyme';
 import { Label } from '@patternfly/react-core';
 
 describe('App tests', () => {
   test('should render default App component', () => {
-    const view = shallow(<Index />);
+    const view = shallow(<Search />);
     expect(view).toMatchSnapshot();
   });
 
   it('should render a search Label', () => {
-    const wrapper = mount(<Index />);
+    const wrapper = mount(<Search />);
```

Comment:
Nice, the Index thing had kind of been bugging me as badly named. I know this is a small thing to call out but I appreciate the small details!